libraries/client/client_webhook.py

```diff
@@ -187,6 +192,12 @@ def process_webhook(self):
             source_url = self.build_multipart_source(file_key, book)
             identifier, job = self.send_job_request_to_tx_manager(commit_id, source_url, rc, repo_name, repo_owner,
                                                                   count=book_count, part=i, book=book)
+
+            # Send lint request to tx-manager
+            lint_results = self.send_lint_request_to_tx_manager(job, source_url)
```

Comment:
But I always do merge the warnings that come back from the linter lambda call with the overall build_log...I'm currently making big changes in my next commit, not using the tx-manager but having the client manage the job better, the build_log better, and invoke the linter lambda.